react/features/base/react/components/native/SlidingView.js

```diff
@@ -264,7 +264,9 @@ export default class SlidingView extends PureComponent<Props, State> {
                     })
                 .start(({ finished }) => {
                     finished && this._mounted && !show
-                        && this.setState({ showOverlay: false });
+                        && this.setState({ showOverlay: false }, () => {
+                            this.forceUpdate();
```

Comment:
Can you please elaborate on why this is needed? What state is inconsistent and in what way?

Reply:
The sliding view "forgets" to remove the tint after it disappears. I found that due to the async nature of the animation, when we click very fast and the very last state update gets applied React somehow decides not to re-invoke render(), even though the state changes.

You can repro the issue by quickly double clicking the chat button on a real device (I reproed it many times on my iPad Pro)